packages/wonder-blocks-link/components/link.js

```diff
@@ -12,7 +13,7 @@ export type SharedProps = {|
     /**
      * Text to appear on the link.
```

Comment:
Might be worth mentioning here in the docs that it can be a string or a Typography element.